hack/get_ci_logs.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+from bs4 import BeautifulSoup
+import requests
+import os
+from urllib.parse import urljoin, urlparse
+
+
+def extract_timestamp_from_url(base_url):
```

Comment:
Could we use pytyping here at least?

Looks like your comments specify types by the function parameters do not.